framework/test/mocha/common/utils/wait_for.js

```diff
@@ -58,7 +58,7 @@ function blockchainReady(retries, timeout, baseUrl, doNotLogRetries, cb) {
 			.then(res => {
 				retries -= 1;
 				res = JSON.parse(res.body);
-				if (!res.data.loaded && retries >= 0) {
+				if (res.data.syncing && retries >= 0) {
```

Comment:
i think it should check like `res.data.syncing === undefined || res.data.syncing === null`?

Reply:
I think this value in the context of tests is always false and never undefined or null